deploy/k8s/charts/trustification/values.schema.yaml

```diff
@@ -18,6 +18,11 @@ properties:
       When using OpenShift, this should be the application domain, when can be evaluated by running the following
       command: `oc -n openshift-ingress-operator get ingresscontrollers.operator.openshift.io default -o jsonpath='{.status.domain}'`.
 
+  ssoServiceName:
```

Comment:
I don't like the name and location of this variable: a) sso is not necessarly keycloak b) Not everyone uses "OCP" or knows that that means. c) I don't think it properly explains what should go into that variable.